test/Tester/App.config

```diff
@@ -46,4 +46,30 @@
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
+  <system.serviceModel>
```

Comment:
sure, I removed it now... it was added automatically when installing the new ServiceBus package.